desc/objectives/getters.py

```diff
@@ -57,14 +59,19 @@ def get_equilibrium_objective(eq, mode="force", normalize=True):
         for minimizing MHD energy.
     normalize : bool
         Whether to normalize units of objective.
+    jac_chunk_size : int, optional
+
 
     Returns
     -------
     objective, ObjectiveFunction
         An objective function with default force balance objectives.
 
     """
-    kwargs = {"eq": eq, "normalize": normalize, "normalize_target": normalize}
+    kwargs = {
+        **{"eq": eq, "normalize": normalize, "normalize_target": normalize},
```

Comment:
This feels like awkward notation. Isn't this equivalent to appending `eq`, `normalize`, and `normalize_target` to the existing `kwargs`?

Reply:
dictionaries don't have an "append" option but you could do `dict.update`

Reply:
I think I will revert this change as for the equilibrium objective, we'd never do blocked anyways, and that is the only place where the underlying jac_chunk_size matters